Source/Core/DolphinWX/ConfigMain.cpp

```diff
@@ -322,6 +322,7 @@ void CConfigMain::InitializeGUILists()
 void CConfigMain::InitializeGUIValues()
 {
 	const SCoreStartupParameter& startup_params = SConfig::GetInstance().m_LocalCoreStartupParameter;
+	bool NTSC = SConfig::GetInstance().m_LocalCoreStartupParameter.bNTSC;
```

Comment:
Checking bNTSC here doesn't make sense: it doesn't have any meaning when there isn't a game running.